worlds/hylics2/Options.py

```diff
@@ -35,14 +48,23 @@ def get_option_name(cls, value: int) -> str:
             return "TV Island"
         return super().get_option_name(value)
 
+
 class ExtraLogic(DefaultOnToggle):
-    """Include some extra items in logic (CHARGE UP, 1x PAPER CUP) to prevent the game from becoming too difficult."""
+    """
+    Include some extra items in logic (CHARGE UP, 1x PAPER CUP) to prevent the game from becoming too difficult.
+    """
     display_name = "Extra Items in Logic"
 
+
 class Hylics2DeathLink(DeathLink):
-    """When you die, everyone dies. The reverse is also true.
+    """
+    When you die, everyone dies. The reverse is also true.
+    
     Note that this also includes death by using the PERISH gesture.
-    Can be toggled via in-game console command "/deathlink"."""
+    
+    Can be toggled via in-game console command "/deathlink".
```

Comment:
I don't know for sure if the tooltip supports `stuff like this`, but maybe you could try surrounding /deathlink with ` and see what it looks like on webhost? I'd check but I'm not at my home computer atm

Reply:
It seems like that doesn't work, unfortunately.